modules/aws/ecs_test.go

```diff
@@ -21,3 +24,39 @@ func TestEcsCluster(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "terratest", *c2.ClusterName)
 }
+
+func TestEcsClusterWithInclude(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	clusterName := "terratest-" + random.UniqueId()
+	tags := []*ecs.Tag{&ecs.Tag{
+		Key:   aws.String("test-tag"),
+		Value: aws.String("hello-world"),
+	}}
+
+	client := NewEcsClient(t, region)
+	c1, err := client.CreateCluster(&ecs.CreateClusterInput{
+		ClusterName: aws.String(clusterName),
+		Tags:        tags,
+	})
+	assert.Nil(t, err)
```

Comment:
NIT: 

```suggestion
	assert.NoError(t, err)
```

This will show the error clearly if it's not `nil`, making debugging easier.